packages/imperative/src/config/cmd/auto-init/handlers/BaseAutoInitHandler.ts

```diff
@@ -153,17 +153,15 @@ export abstract class BaseAutoInitHandler implements ICommandHandler {
             if (params.arguments.forSure && params.arguments.forSure === true) {
                 // Clear layer, merge, generate schema, and save
                 ImperativeConfig.instance.config.api.layers.set(profileConfig);
-                const schema = ConfigSchema.buildSchema(ImperativeConfig.instance.loadedConfig.profiles);
-                ImperativeConfig.instance.config.setSchema(schema);
+                ConfigSchema.updateSchema();
                 await ImperativeConfig.instance.config.save(false);
             }
         } else {
             // Merge, generate schema, and save
             ImperativeConfig.instance.config.api.layers.merge(profileConfig);
             if (ImperativeConfig.instance.config.api.layers.get().properties.$schema == null) {
                 // TODO What condition should we use to decide whether to (re)generate schema?
-                const schema = ConfigSchema.buildSchema(ImperativeConfig.instance.loadedConfig.profiles);
-                ImperativeConfig.instance.config.setSchema(schema);
+                ConfigSchema.updateSchema();
```

Comment:
After creating the `ConfigSchema.updateSchema` API, all occurrences of `buildSchema + setSchema` were updated to use it. 😋